src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -1465,9 +1618,14 @@ await DownloadFileAsync(file.FilePath, destination, cancellationToken)
         /// <inheritdoc />
         public Task UnpinChatMessageAsync(
             ChatId chatId,
-            CancellationToken cancellationToken = default
+            CancellationToken cancellationToken = default,
+            int messageId = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.

Reply:
Changing parameter order is considered breaking change, so it will happen in next major release